modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -120,9 +132,16 @@ request-string=Request String
 response-string=Response String
 scope-parameter-name=Scope Parameter Name
 search-...=Search...
+search-container=Search Container
+search-container-help=Access helper methods to retrieve Search Container values like the total results count.
```

Comment:
This is okay as is, but I might try to put a little separation between the functionality and the example:

> Access helper methods to retrieve Search container values (for example, total results count).

Reply:
Should we capitalize both words in `Search Container`? (Or no capitalization?)

Reply:
Good question. I intended to capitalize both words, but only because that's what was in the original key. I expect that we should not capitalize it at all.